### PR TITLE
Add unit tests for home

### PR DESCRIPTION
**Pull request by ctsstc**
Implements unit tests for Home 

## Related Issues
- #2

### CODE REVIEW

File: tests/unit/home.spec.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { mount } from '@vue/test-utils';
+import Home from '@/views/Home.vue';
+
+describe('Home', () => {
+  const wrapper = mount(Home);
+
+  it('should contain an anchor tag that links to google', () => {
```
Comment (Otis0620) [inline tests/unit/home.spec.ts]: I feel like you are testing the external link component, which already tests for an anchor tag. I think you're testing outside the scope of this component.

File: tests/unit/home.spec.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    const anchor = wrapper.find('a');
+    const href = anchor.attributes('href');
```
Comment (Otis0620) [inline tests/unit/home.spec.ts]: Could use some white space after href

File: tests/unit/home.spec.ts (hunk 1 above)
Comment (ctsstc) [reply tests/unit/home.spec.ts]: I had that same feeling as well, it actually does have coverage overlap with the External Link component.

I'm not sure what else to test for here, unless I can/should be directly testing the data on the export?

File: tests/unit/home.spec.ts (hunk 2 above)
Comment (ctsstc) [reply tests/unit/home.spec.ts]: Read my mind, there's also a few other spots I was tempted to put a commit in for whitespace. Sometimes I go with hyper condensed tests, I should see a doctor about this.

File: tests/unit/home.spec.ts (hunk 1 above)
Comment (Otis0620) [reply tests/unit/home.spec.ts]: So you might test that it contains the h1 text because if the component does not render properly then it would not show the text

Comment (ctsstc) [reply tests/unit/home.spec.ts]: Good point, I was going off of coverage which seems to only inspect the javascript for the component. We need to keep something to test that there's a link to google to keep that coverage.

File: tests/unit/home.spec.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    const div = wrapper.find('div.home');
+
+    it('should contain an h1 with home page', () => {
+      const h1 = div.find('h1');
+      const text = h1.text();
+
+      expect(text).toContain('home page');
+    });
+
+    it('should contain an anchor tag that links to google', () => {
```
Comment (Otis0620) [inline tests/unit/home.spec.ts]: I think this test could be omitted

File: tests/unit/home.spec.ts (hunk 1 above)
Comment (Otis0620) [reply tests/unit/home.spec.ts]: @drewjbartlett how would you test this component?

**Comment by ctsstc** [comment]
TLDR; 
image

File: tests/unit/home.spec.ts (hunk 3 above)
Comment (drewjbartlett) [reply tests/unit/home.spec.ts]: Agreed. A snapshot test could cover all of these :D
